vyper/exceptions.py

```diff
@@ -51,74 +51,117 @@ def __str__(self):
 
 class PythonSyntaxException(ParserException):
     """
-    Conversion from error using ast.parse()
+    Invalid syntax.
+
+    This error is converted from errors raised during ast.parse
     """
     def __init__(self, syntax_error: SyntaxError, source_code: str):
         item = types.SimpleNamespace()  # TODO: Create an actual object for this
         item.lineno = syntax_error.lineno
         item.col_offset = syntax_error.offset
-        item.source_code = source_code
+        item.full_source_code = source_code
         super().__init__(message=f'SyntaxError: {syntax_error.msg}', item=item)
 
 
-class VariableDeclarationException(ParserException):
-    pass
+class SyntaxException(ParserException):
+    """Valid Python syntax, but invalid Vyper syntax."""
```

Comment:
Does it make sense to have two types of syntax error? We might consider merging `PythonSyntaxException` with `SyntaxException`.

Reply:
I agree with this. You can modify the wording of a Python syntax exception so that it's just a normal syntax exception (using raise from)